web/src/app/help/help.component.scss

```diff
@@ -1,3 +1,7 @@
+.content {
+    font-family: 'Noto Serif SC', 'Noto Serif TC', 'Ext-0', 'Ext-1', serif;
```

Comment:
can we make this a global font style? (i.e. create a new class in web/src/styles.scss and reuse the class in the help component)

Reply:
Yea I was thinking about it but I kind of get confused by the the font settings in all the places... I guess we can talk about this this Sunday?

Reply:
In general, we should limit the number of font styles in the app. So it makes sense to move all font styles into a centralised place.

Can you create an issue for this?